native/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Super POM included by all other sub-POM(s)-->
```

Comment:
minor: Might want to remove this comment. I'm guessing it was copy/paste from parent pom.